refl1d/probe.py

```diff
@@ -1725,36 +1730,39 @@ def shared_beam(self, intensity=1, background=0,
                 x.sample_broadening = sample_broadening
 
     def oversample(self, n=6, seed=1):
+        self._oversample(n, seed)
+        self.oversampling = n
+        self.oversampling_seed = seed
```

Comment:
Is this going to conflict with the theta offset branch?

Maybe we should revert this file on this branch since it is limited to oversampling changes.

Reply:
This does not conflict with the theta_offset branch.  It is the same change made in that branch, so they merge just fine.  Could probably revert it as you say with no issues.